pkg/hardwareutils/bmc/idrac_virtualmedia.go

```diff
@@ -85,7 +85,7 @@ func (a *redfishiDracVirtualMediaAccessDetails) PowerInterface() string {
 }
 
 func (a *redfishiDracVirtualMediaAccessDetails) RAIDInterface() string {
-	return "no-raid"
+	return "idrac-redfish"
```

Comment:
This was enabled upstream on the grounds that it's already enabled for the idrac driver, but in fact we have disabled it downstream: https://github.com/openshift/baremetal-operator/blob/master/pkg/hardwareutils/bmc/idrac.go#L94-L96
@dtantsur I assume we want to do the same here.

Reply:
I guess for consistency it makes sense, yes. We should look into re-enabling it since there is some interest to experiment with RAID.

Reply:
https://github.com/openshift/baremetal-operator/pull/219